packages/extension-api-explorer/src/ExtensionApiExplorer.tsx

```diff
@@ -87,11 +87,14 @@ export const ExtensionApiExplorer: FC = () => {
     if (sdk && !specs) loadSpecs().catch((err) => console.error(err))
   }, [specs, sdk])
 
-  const extensionAdaptor = new ExtensionAdaptor(getExtensionSDK())
+  const extensionAdaptor = new ExtensionAdaptor(
+    getExtensionSDK(),
+    extensionContext.core40SDK
```

Comment:
As discussed, we'll need to reconcile 3.1 API calls vs 4.0 API calls. It would be great to include @bryans99 in that conversation.